.github/workflows/upgrade-python-requirements.yml

```diff
@@ -3,20 +3,20 @@ name: Upgrade Requirements
 on:
   schedule:
     # will start the job at 13:15 UTC on 1st of every month
```

Comment:
nit: if you are going to change the cron schedule, can you update the comment as well?